### PR TITLE
Add paper aos and submit aos for all caseworkers

### PR DESCRIPTION
**Pull request by jacmurphy**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DIV-6302

### Change description ###

Add AddPaperAOS
Update solicitorSubmitAOS from "R" to "CURD"

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: definitions/divorce/json/CaseEvent.json [hunk 1]
```diff
@@ -2755,5 +2755,15 @@
     "PreConditionState(s)": "*",
     "PostConditionState": "*",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "04/05/2020",
```
Comment (jacmurphy) [inline definitions/divorce/json/CaseEvent.json:2760]: Is this a suitable date?

File: definitions/divorce/json/AuthorisationCaseEvent.json [hunk 2]
```diff
@@ -5289,7 +5289,7 @@
     "CaseTypeID": "DIVORCE",
     "CaseEventID": "solicitorSubmitAOS",
     "UserRole": "caseworker-divorce-courtadmin_beta",
-    "CRUD": "R"
+    "CRUD": "CRUD"
   },
```
Comment (jacmurphy) [inline definitions/divorce/json/AuthorisationCaseEvent.json:5293]: There are many AOS events currently present with caseworker_beta set to "R". 
Is this event all that is required for this story?
@yarpo

File: definitions/divorce/json/CaseEvent.json (hunk 1 above)
Comment (yarpo) [reply definitions/divorce/json/CaseEvent.json:2760]: as long as it's not in the past it's fine

File: definitions/divorce/json/AuthorisationCaseEvent.json (hunk 2 above)
Comment (yarpo) [reply definitions/divorce/json/AuthorisationCaseEvent.json:5293]: I have no idea. We need to use the event that will be triggered for AOS pack 2. I have never worked or refined stories about it :( [or at least I don't remember event's name)

Comment (jvictorcf) [reply definitions/divorce/json/AuthorisationCaseEvent.json:5293]: I think your story wants you to do them all, @jacmurphy.
@yarpo, not sure we want to release this until the very end of the epic.

Comment (jacmurphy) [reply definitions/divorce/json/AuthorisationCaseEvent.json:5293]: Thanks @jvictorcf . Ill update them now

**Comment by yarpo** [comment]
The ticket is withdrawn. I suppose it can be closed.

Please re-open if I'm wrong.

**Pull request closed without merging**